File: YH_Toast/YH_Toast/ViewController.m
//
//  ViewController.m
//  YH_Toast
//
//  Created by 余晗 on 16/1/7.
//  Copyright © 2016年 余晗. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
//    [YH_Toast showToastFromStr:@"吃饭了木有？"];
    UIButton *btn=[UIButton buttonWithType:UIButtonTypeCustom];
    [btn setFrame:CGRectMake((self.view.frame.size.width-40)/2, (self.view.frame.size.height-40)/2, 40, 40)];
    [btn addTarget:self action:@selector(popToast) forControlEvents:UIControlEventTouchUpInside];
    [btn setTitleColor:[UIColor blackColor] forState:0];
    [btn setTitle:@"click" forState:0];
    [self.view addSubview:btn];
    // Do any additional setup after loading the view, typically from a nib.
}
-(void)popToast
{
     [YH_Toast showToastFromStr:@"吃饭了木有？"];
}
- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: YH_Toast_swift/YH_Toast_swift/ViewController.swift
//
//  ViewController.swift
//  YH_Toast_swift
//
//  Created by 余晗 on 1/11/16.
//  Copyright © 2016 yuhan. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    var btn : UIButton = UIButton();
    override func viewDidLoad() {
        super.viewDidLoad()
       
        btn.frame=CGRectMake(0, 0, view.frame.width, 100);
        btn.setTitleColor(UIColor.blackColor(), forState: UIControlState.Normal);
        btn.setTitle("点我", forState: UIControlState.Normal);
        btn.addTarget(self, action:Selector("clickThisBtn:"), forControlEvents: UIControlEvents.TouchUpInside);
        view.addSubview(btn);
        // Do any additional setup after loading the view, typically from a nib.
    }

    func clickThisBtn(btn :UIButton)
    {
        NSLog("click");

        YH_Toast().showToastFromStr(btn.titleForState(UIControlState.Normal)!);
        
    }
    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

